site/src/routes/_components/WhosUsingSvelte.svelte

```diff
@@ -1,10 +1,30 @@
 <script>
-	import { companies } from './WhosUsingSvelte.js';
-	const randomizer = ({prominent}) =>  Math.random();
+	import { companies } from "./WhosUsingSvelte.js";
```

Comment:
we use single quotes for all strings
```suggestion
	import { companies } from './WhosUsingSvelte.js';
```